### PR TITLE
feat:split user and account

### PR DESCRIPTION
**Pull request by xiao-jay**
Signed-off-by: 晓杰 <2561589453@qq.com>
#1950 

split account and user , Easy to iterate independently
- [x] test payment and recharge
- [x] test metering modele (metering e2e test)
- [x] test user (看user下面的kubeconfig有没有生成)
- [x] fix metering error

### test user
```
make build run
kubectl apply -f  config/sample/user_v1_user.yaml 
```

@cuisongliu user is ok.
```
➜  samples git:(split-user-account) kubectl get user f8699ded-58d3-432b-a9ff-56568b57a38d  -oyaml
apiVersion: user.sealos.io/v1
kind: User
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"user.sealos.io/v1","kind":"User","metadata":{"annotations":{},"name":"f8699ded-58d3-432b-a9ff-56568b57a38d"},"spec":{"csrExpirationSeconds":1000000000}}
  creationTimestamp: "2023-02-07T03:57:23Z"
  finalizers:
  - sealos.io/user.finalizers
  generation: 1
  name: f8699ded-58d3-432b-a9ff-56568b57a38d
  resourceVersion: "2563181"
  uid: 2fcb9130-12f7-487d-a347-8762ea160b7d
spec:
  csrExpirationSeconds: 1000000000
status:
  conditions:
  - lastHeartbeatTime: "2023-02-07T03:57:23Z"
    lastTransitionTime: "2023-02-07T03:57:23Z"
    message: user has been initialized
    reason: Initialized
    status: "True"
    type: Initialized
  - lastHeartbeatTime: "2023-02-07T03:57:23Z"
    lastTransitionTime: "2023-02-07T03:57:23Z"
    message: sync kube config successfully
    reason: Ready
    status: "True"
    type: KubeConfigSyncReady
  - lastHeartbeatTime: "2023-02-07T03:57:24Z"
    lastTransitionTime: "2023-02-07T03:57:24Z"
    message: sync owner ug successfully
    reason: Ready
    status: "True"
    type: OwnerUGSyncReady
  - lastHeartbeatTime: "2023-02-07T03:57:24Z"
    lastTransitionTime: "2023-02-07T03:57:24Z"
    message: sync owner ug namespace binding successfully
    reason: Ready
    status: "True"
    type: OwnerUGNamespaceBindingSyncReady
  - lastHeartbeatTime: "2023-02-07T03:57:24Z"
    lastTransitionTime: "2023-02-07T03:57:24Z"
    message: User is available now
    reason: Ready
    status: "True"
    type: Ready
  kubeConfig: |
    apiVersion: v1
    clusters:
    - cluster:
        certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCakNDQWU2Z0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwdGFXNXAKYTNWaVpVTkJNQjRYRFRJeU1Ea3hNREE1TVRJek9Gb1hEVE15TURrd09EQTVNVEl6T0Zvd0ZURVRNQkVHQTFVRQpBeE1LYldsdWFXdDFZbVZEUVRDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBSnlxCmduV3NnYmlWejJWMGRoR3U3L2UzcW9pcC9lem4rZ3I3RHNTejVNaFh4QzB4Zk5lUzRMYVYrMDN2S3g1VEdJKzQKZnVEYkJ4aUtZd1Y1cU5WRS8vYmNjRm16eEVYckdjVCtDemQyc2lSS2ZTZThDa3NNK3h1MnhFd3BtWlBTMTk0cwppaHJUTytJcitieWZuTXJQWjlLZzVkcFN3cUpXTlFPKzJzZlp4TkozVUVDcEViYVQ4U1A4anFXOXRRWGhNbWs0ClNkcG9FY09sUWJCRjloYkRoYkZNWDA5V01KeGtrdjVpTmF5RlA1RUp6a0xHT2x1bzU4b1BjdythQ3AwMXM5YnYKdlZJQ3ZqaDlnZ0JtVjROTFlPQ0k5N3J1VVh3QWpRckw3bk5taGExd2NPcnRjb05RYndWQzZkK2tNMWpuVksydwp1RWlYalVheUI4ekoyYVRrZXFrQ0F3RUFBYU5oTUY4d0RnWURWUjBQQVFIL0JBUURBZ0trTUIwR0ExVWRKUVFXCk1CUUdDQ3NHQVFVRkJ3TUNCZ2dyQmdFRkJRY0RBVEFQQmdOVkhSTUJBZjhFQlRBREFRSC9NQjBHQTFVZERnUVcKQkJTZk5uamRGeTNLdDhzS3RSa2NzU2JneWwra01EQU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFKUUpiS2swZwpNNlRYTTZHUGVOSlJoZzZJYXROMUNDWTZGd2JlVHNYano5TmhqQVdQUVBDSVVVdDZvVUkxUjE2M1JmTk1QZUljCklwcUpwaXBpSmlCT1Z0WGlVb2NHMnpzOEhJc2RuNFI5Z0NjS3dOV0R0RUxxVk9BS0swOGxGdXhZS2ZiZVlsQ3oKemdVLzI0RzRwVFNRd0RyL1phUHpzNEl2YzE2N3RoZVFzWDFvYXV2MERDM1RxOVJFTzJkancxbG1HUWpWdS9XRwppM1lyUmZTU0hHNzBuQUEySlVPZXBtcENqamJER2Q5cUpRRnduSkRScmJIRXFCYysrVkhjV0g2UXJsdW50WnpzCmNRaDZldEdmQ2RZb04xM3l0VG1LNTFpN3dxRFBLTVM4aHZFRWdIRkxmZjFtQVVvRG9GWWVyLzZTQXF6S3I2Qm4KNDNNazFLLzlMQURwZXc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
        server: https://127.0.0.1:61792
      name: sealos
    contexts:
    - context:
        cluster: sealos
        namespace: ns-f8699ded-58d3-432b-a9ff-56568b57a38d
        user: f8699ded-58d3-432b-a9ff-56568b57a38d
      name: f8699ded-58d3-432b-a9ff-56568b57a38d@sealos
    current-context: f8699ded-58d3-432b-a9ff-56568b57a38d@sealos
    kind: Config
    preferences: {}
    users:
    - name: f8699ded-58d3-432b-a9ff-56568b57a38d
      user:
        token: eyJhbGciOiJSUzI1NiIsImtpZCI6IjZvZ25EOHFXZ25nWFl4Tzc5TGdwMmlfWmVFR1N6VF91SG5PNF8wRnoyYkUifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImY4Njk5ZGVkLTU4ZDMtNDMyYi1hOWZmLTU2NTY4YjU3YTM4ZC1nZW5lcmF0b3ItdG9rZW4iLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoiZjg2OTlkZWQtNThkMy00MzJiLWE5ZmYtNTY1NjhiNTdhMzhkIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZXJ2aWNlLWFjY291bnQudWlkIjoiZTMzNzI3ZjEtNWJjMy00NmNmLWI3ZjktZDEzZWU2MTRlZGZlIiwic3ViIjoic3lzdGVtOnNlcnZpY2VhY2NvdW50OmRlZmF1bHQ6Zjg2OTlkZWQtNThkMy00MzJiLWE5ZmYtNTY1NjhiNTdhMzhkIn0.AT87PYi5OvoAJRLWulbBLFyCq2LmoA54JqApPspEMqV8iNsUtwhrfwKbrUysxbfU_b6u3_amGdBpiaLFU5cYIT81uiAYWkVb6cNtHadmucafWF4cOu8K2zSTHCuUg3u3EK3OikOB-2xx9IGC3mcK7UPfzu8oR91nYNZjElR1em0hSxT8okugMSurXJ8iux_s9_dEgOU7un0CDTrRox_vg5i855WzecUr7SwjKcFBPguwiTzmZ89oUp0joiTTipoEOsPBkWhs05u1FOExL_cV2yoB-JKxAq980n6mCvgCEg64u7893blFomgsOXg88aLqaP-xL_SWyzyO7kylR4WP3A
  observedCSRExpirationSeconds: 1000000000
  observedGeneration: 1
  phase: Active

```

### CODE REVIEW

File: .github/workflows/controllers.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        module: [ user, terminal, metering, infra, cluster, app, imagehub]
+        module: [ user, terminal, metering, infra, cluster, app, imagehub,account]
```
Comment (LeezQ) [inline .github/workflows/controllers.yml:21]: need to format code

Comment (xiao-jay) [reply .github/workflows/controllers.yml]: fine

File: controllers/account/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+### 部署方式
+```
+先提前修改image: controller:latest,修改为自己打的镜像
```
Comment (cuisongliu) [inline controllers/account/README.md:3]: IMG=XXX  make pre-deploy

File: controllers/account/Makefile [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
+	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
+
+.PHONY: deploy
+deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default | kubectl apply -f -
+
+.PHONY: pre-deploy
+pre-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```
Comment (cuisongliu) [inline controllers/account/Makefile:100]: add Kubefile to deploy/

```
FROM scratch
USER 65532:65532
COPY registry ./registry
COPY manifests ./account-controller
CMD ["kubectl apply -f account-controller/deploy.yaml"]

```

File: controllers/account/README.md (hunk 2 above)
Comment (xiao-jay) [reply controllers/account/README.md]: done

File: controllers/account/Makefile (hunk 3 above)
Comment (xiao-jay) [reply controllers/account/Makefile:100]: I see user and metering module deploy file is named dockerfile,maybe named dockerfile is better.

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**